src/components/customers/CustomerSettings.tsx

```diff
@@ -281,6 +300,90 @@ export const CustomerSettings = ({ customerId }: CustomerSettingsProps) => {
         )}
       </InfoBlock>
 
+      <InlineSectionTitle>
+        <Typography variant="subhead" color="grey700">
+          {translate('text_63ea0f84f400488553caa765')}
+        </Typography>
+        {typeof customer?.billingConfiguration?.documentLocale !== 'string' ? (
+          <Button
+            disabled={loading}
+            variant="quaternary"
+            endIcon={isPremium ? undefined : 'sparkles'}
+            onClick={() =>
+              isPremium
+                ? editCustomerDocumentLocale?.current?.openDialog()
+                : premiumWarningDialogRef.current?.openDialog()
+            }
+          >
+            {translate('text_63ea0f84f400488553caa761')}
+          </Button>
+        ) : (
+          <Popper
+            PopperProps={{ placement: 'bottom-end' }}
+            opener={<Button disabled={loading} icon="dots-horizontal" variant="quaternary" />}
+          >
+            {({ closePopper }) => (
+              <MenuPopper>
+                <Button
+                  disabled={loading}
+                  startIcon="pen"
+                  variant="quaternary"
+                  align="left"
+                  onClick={() => {
+                    editCustomerDocumentLocale.current?.openDialog()
+                    closePopper()
+                  }}
+                >
+                  {translate('text_63ea0f84f400488553caa785')}
+                </Button>
+                <Button
+                  disabled={loading}
+                  startIcon="trash"
+                  variant="quaternary"
+                  align="left"
+                  onClick={() => {
+                    deleteCustomerDocumentLocale.current?.openDialog()
+                    closePopper()
+                  }}
+                >
+                  {translate('text_63ea0f84f400488553caa786')}
+                </Button>
+              </MenuPopper>
+            )}
+          </Popper>
+        )}
+      </InlineSectionTitle>
+
+      <InfoBlock>
+        {loading ? (
+          <>
+            <Skeleton variant="text" width={320} height={12} marginBottom={theme.spacing(4)} />
+            <Skeleton variant="text" width={160} height={12} />
+          </>
+        ) : (
+          <>
+            <Typography variant="body" color="grey700">
+              {typeof customer?.billingConfiguration?.documentLocale === 'string'
+                ? // @ts-ignore
```

Comment:
Why do you need to `ts-ignore` here ?

Reply:
documentLocales is now typed, all ignore have been removed in last commit